modules/home-manager/kvantum.nix

```
@@ -0,0 +1,43 @@
+{ config
+, lib
+, pkgs
+, ...
+}:
+let
+  cfg = config.qt.catppuccin-kvantum;
```

```suggestion
  cfg = config.qt.style.catppuccin;
```
this is more inline with the option we're actually modifying here as well as our other options